src/components/global/Playground/index.tsx

```diff
@@ -194,8 +258,19 @@ export default function Playground({
                 </svg>
               </a>
             </Tippy>
-            <Tippy theme="playground" arrow={false} placement="bottom" content="Copy source code">
-              <button className="playground__icon-button playground__icon-button--primary" onClick={copySourceCode}>
+            <Tippy
+              theme="playground"
+              arrow={false}
+              placement="bottom"
+              content={hasOutputTargetOptions ? 'Unavailable in multi-file examples' : 'Copy source code'}
```

Comment:
~ Could we also mention somehow that you should use the copy buttons on the individual code blocks? Maybe "For multi-file examples, use copy buttons on code blocks" or something.

Reply:
Yeah, I like that a lot!